pkg/crc/machine/machine.go

```diff
@@ -116,7 +115,7 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		}
 
 		if s != state.Running {
-			logging.InfoF(" Starting stopped VM ...")
+			logging.InfoF("Starting stopped VM ...")
```

Comment:
This is just fixing the offset but you will still see the message twice.

```
INFO Starting stopped VM ...                      
INFO Starting stopped VM ...                     
```

Reply:
There was another print line which was causing the message to print twice, which is removed https://github.com/code-ready/crc/pull/198/files#r291927668